data-structures/ds-stk/c/stack implementation using link list.c

```diff
@@ -0,0 +1,52 @@
+#include<stdio.h>
+#include<stdlib.h>
+struct linkListNode{
+	int data;
+	struct linkListNode *next;
+};
+typedef struct linkListNode *node;
+
+node start=NULL;
+
+node createNode(){
+	node newNode;
+	newNode=(node)malloc(sizeof(node));
```

Comment:
Codacy found an issue: [Size of pointer 'newNode' used instead of size of its data.](https://app.codacy.com/gh/FOSS-UCSC/FOSSALGO/pullRequest?prid=6274877)